src/main/java/com/axibase/tsd/client/SelfSignedTrustManager.java

```diff
@@ -1,18 +1,21 @@
 package com.axibase.tsd.client;
 
+import lombok.extern.slf4j.Slf4j;
+
 import javax.net.ssl.TrustManager;
 import javax.net.ssl.TrustManagerFactory;
 import javax.net.ssl.X509TrustManager;
+import java.io.Serializable;
 import java.security.KeyStore;
 import java.security.KeyStoreException;
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
 
+@Slf4j
+public class SelfSignedTrustManager implements X509TrustManager, Serializable, Cloneable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "clone()" method to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.shmagrinsky.com/coding_rules#rule_key=squid%3AS2157)